### ISSUE TITLE
Trying to build your project but more error!.

### ISSUE THREAD

**Issue by tuanteku9x** [keowu/pefixerextremenet#2]
Hi pro, I'm trying to build your project with visual studio 2022, but I got much more errors in the "error list". Sorry, I'm not a pro coder 
Here screenshot: https://imgur.com/a/DUykoWX

**Comment by keowu** [comment]
Hello, thank you very much for your issue, I analyzed the image it looks like a project configuration error, as soon as I have some time I will record a video explaining how to compile and start the debug to be able to use it while it is in development, soon (as soon as I have some time) I'll post the link to the explainer video here and I'll update the repository as well.

**Comment by tuanteku9x** [comment]
> Hello, thank you very much for your issue, I analyzed the image it looks like a project configuration error, as soon as I have some time I will record a video explaining how to compile and start the debug to be able to use it while it is in development, soon (as soon as I have some time) I'll post the link to the explainer video here and I'll update the repository as well.

Hi pro, is your project fixed for my report feeback?

**Comment by keowu** [comment]
yes i have a solution to help you with the problem, i was planning to do a setup video but lately i am not having much time so i will explain you how you can fix this problem:

1 - Make sure your visual studio is version 143(2022)
2 - Get the project via git
3 - After you download the project open it normally to be able to set the project to developer debug mode
4 - Change your project to the following global setting "debug - x86" to be able to develop.
5 - With your project context open right click on it go to properties -> C/C++ -> Language, look for Language Standard set to "ISO Standard C++20 (/std:c++20)" or "Preview - C++ Latest Working Draft Resources (/std:c++latest)", apply this setting.
6 - When trying to compile if step 4 has been done wrongly you will receive an error linking curl, because its pre-build was made for the development environment at that moment, but if everything is right your build will work fine.

I have some information and warnings, this version of the software is not yet complete, it has several bugs, and I am working to fix most of them as I have time and also develop their functions, if you fix any bugs or find any new bugs you can fix them and request a merge with your changes, I recommend that you proceed only in debug mode, using breakpoints, and definitely for now don't try to fix a protected binary as this project is for study only and may not work well at the moment.

hope this helps you.
